nservicebus/persistence/authoring-custom.md

```diff
@@ -125,8 +125,13 @@ In order to provide a custom timeout persister implementation:
 - In versions 4.x (starting from 4.4) and 5.x you need to implement interfaces `IPersistTimeouts` and `IPersistTimeoutsV2`. 
     - The reference in-memory implementation of timeouts persistence for NServiceBus v4.x can be seen [here](https://github.com/Particular/NServiceBus/blob/support-4.4/src/NServiceBus.Core/Persistence/InMemory/TimeoutPersister/InMemoryTimeoutPersistence.cs). 
     - The reference in-memory implementation of timeouts persistence for NServiceBus v5.x can be seen [here](https://github.com/Particular/NServiceBus/blob/support-5.0/src/NServiceBus.Core/Persistence/InMemory/TimeoutPersister/InMemoryTimeoutPersister.cs).
- 
-The interface `IPersistTimeoutsV2` was introduced to prevent a potential message loss, while the `IPersistTimeouts` interface allows to maintain backwards compatibility (you can find more details in the following [issue description](https://github.com/Particular/NServiceBus/issues/2885)).
+    
+   **Note:** The interface `IPersistTimeoutsV2` was introduced to prevent a potential message loss, while the `IPersistTimeouts` interface allows to maintain backwards compatibility (you can find more details in the following [issue description](https://github.com/Particular/NServiceBus/issues/2885)).
+
+- Starting from version 6.0 you need to implement interfaces `IPersistTimeouts` and `IQueryTimeouts`.
+	- The reference in-memory implementation of timeouts persistence for version 6.0 can be seen [here](https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Persistence/InMemory/TimeoutPersister/InMemoryTimeoutPersister.cs).
+    
+   **Note:** The interface `IQueryTimeouts` has been extracted from `IPersistTimeouts` in version 6.0 in order to explicitly separate responsibilities.
```

Comment:
does it make to "repeat" the interface splitting in it's own note? I like the SoC argument, but couldn't we just add to the paragraph above?